.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: RELEASE
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
Ideally this should be configured on _tags_.

Nice workflow I have setup in other packages i that:
1. _master_ build checks wether version is bumped in `package.json` If it's bumped then tag is pushed to repository
2. _tag_ build publishes new version to npm

You may check how that's achieved e.g. here: https://github.com/serverless/utils/tree/master/.github/workflows. It is also documented here: https://github.com/serverless/utils/blob/master/RELEASE_PROCESS.md

Still I think it's totally fine if in scope of this PR we just focus on PR's and _master_ validation (no publishing)

Reply:
yes, using tags validation directly is much cleaner workflow, I have added tag validation instead of branch from example